web/cobrands/shropshire/assets.js

```diff
@@ -45,16 +45,35 @@ fixmystreet.assets.add(defaults, {
     stylemap: new OpenLayers.StyleMap({
          'default': highways_style
      }),
-    always_visible: true,
-    non_interactive: true,
     usrn: {
         attribute: 'USRN',
         field: 'site_code'
     },
     road: true,
     asset_item: 'road',
     asset_type: 'road',
+    no_asset_msg_id: '#js-not-a-road',
+    always_visible: true,
+    non_interactive: true,
     all_categories: true,
+    actions: {
+        found: function(layer, asset) {
+            fixmystreet.message_controller.road_found(layer, asset.attributes.SITE_CLASS, function(name)
```

Comment:
No biggie, but to be consistent with the rest of this file and the other JS, we put `{` on the same line as the function/if/else statements, and therefore with less indenting.